### PR TITLE
Updating/clarifying template explanation

### PR DESCRIPTION
**Pull request by heymajor**
This change only affects documentation. Updating/clarifying templates and use cases. A broken link was removed, and the info was re-purposed. 

I appreciate this code being available.

Super nervous about this/hope I didn't cross any lines and followed the requested process - my first pull request ;)

### CODE REVIEW

File: docs/recipes.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ jrnl @alberto --tags | grep @melo
 
 And will get something like `@melo: 9`, meaning there are 9 entries
 where both `@alberto` and `@melo` are tagged. How does this work? First,
-`jrnl @alberto` will filter the journal to only entries containing the
+`jrnl @alberto` will filter the journal to only entries containing **the**
```
Comment (micahellison) [inline docs/recipes.md]: Why bold "the"?

File: docs/recipes.md [hunk 2]
```diff
@@ -70,17 +70,61 @@ jrnlimport () {
...
+template: "/path/to/template.txt"
+timeformat: '%Y-%m-%d %H:%M'
+version: "2.1.1"
+```
+
+Changes can be saved as you continue writing the journal entry and will be
+logged as a new entry in the journal you specified in the original argument.
+
+!!! tip 
+To read your journal entry or to verify the entry saved, you can use this
```
Comment (micahellison) [inline docs/recipes.md]: This needs indenting to register as a tip

File: docs/recipes.md [hunk 3]
```diff
@@ -70,17 +70,61 @@ jrnlimport () {
...
+timeformat: '%Y-%m-%d %H:%M'
+version: "2.1.1"
+```
+
+Changes can be saved as you continue writing the journal entry and will be
+logged as a new entry in the journal you specified in the original argument.
+
+!!! tip 
+To read your journal entry or to verify the entry saved, you can use this 
+command: `jrnl -n 1' (Check out [Import and Export](../export/#export-to-files) for more export options).
```
Comment (micahellison) [inline docs/recipes.md]: Code formatting should be closed with a backtick \` instead of an apostrophe '

File: docs/recipes.md [hunk 4]
```diff
@@ -107,10 +156,11 @@ jrnl -on "$(jrnl --short | shuf -n 1 | cut -d' ' -f1,2)"
...
 
-To use external editors for writing and editing journal entries, set
-them up in your `jrnl.yaml` (see `advanced usage <advanced>` for
-details). Generally, after writing an entry, you will have to save and
-close the file to save the changes to jrnl.
+Configure your preferred external editor by updating the `editor` option 
+in your `jrnl.yaml` file. (See [advanced usage](../advanced) for details). 
+
+!!! note
+    To save and log any entry edits, save and close the file.
```
Comment (micahellison) [inline docs/recipes.md]: Nice touch on changing the tense and clarifying this.

File: docs/recipes.md [hunk 5]
```diff
@@ -70,17 +70,61 @@ jrnlimport () {
...
+A more efficient way to work with a template file is to declare the file
+in your config file by changing the `template` setting from `false` to the
+template file's path in double quotes:
+
+```sh
+...
+tagsymbols: "@"
+template: "/path/to/template.txt"
+timeformat: '%Y-%m-%d %H:%M'
+version: "2.1.1"
```
Comment (micahellison) [inline docs/recipes.md]: What do you think about just showing the "template" field here? I'm of two minds - it's nice to show the neighboring fields for context, but also, I'm afraid that leaving the version field in here may be confusing in the long run.

**Comment by heymajor** [comment]
Thank you for your questions and pointing out the errors. 

I realized that I could edit the doc and make the changes you were commenting on. After you comment, should I respond to each/resolve the comment/make the changes, respond to each/leave comments open/don't make changes, or respond to complex comments/close out smaller changes after their made?

**Comment by micahellison** [comment]
This looks great! Thank you for your work on this.

**Pull request merged**